docs/wxm-devices/wifi-m5-bundle/m5-troubleshooting.mdx

```diff
@@ -174,6 +174,56 @@ d. Top cap in white is the rest of M5 Miner.
 </div>
 </details>
 
+
+## Other Issues
+
+### M5 not working in 5 GHz WiFi.
+M5 is designed to work **only** at the 2.4GHz frequency. Please make sure the 2.4Ghz band on your router is enabled. If you are still experiencing issues, please contact us through [support](https://support.weatherxm.com)
+
+<hr />
```

Comment:
This should be moved above under the WiFi Connectivity section. Also add a dot at the end please.